ddtrace/__init__.py

```diff
@@ -5,6 +5,11 @@
 
 LOADED_MODULES = frozenset(sys.modules.keys())
 
+# Configuration for the whole tracer from file. Do it before anything else happens.
+from ddtrace.internal.native import _apply_configuration_from_disk
+
+_apply_configuration_from_disk()
```

Comment:
Maybe a comment/todo mentioning this is not a good way of applying configuration and we plan to integrate better with the rest of the library?

Reply:
You mean https://github.com/DataDog/dd-trace-py/pull/12260/files#diff-73926400eb85d675f94d449f80de5ee4d698fac2e6ea38e53b1f01d27cb729c7R28-R29?